TestProjects/Eventhooks/Eventhooks.Droid/Eventhooks.Droid.csproj

```diff
@@ -50,7 +50,6 @@
     <Compile Include="..\..\..\AssemblyInfo.cs">
       <Link>Properties\AssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
It's part of the PR requirement `All projects build`. I think this was just forgotten earlier.